src/js/services/app-service.js

```diff
@@ -0,0 +1,34 @@
+import { store } from 'stores/app-store';
```

Comment:
What's the purpose of a service? Doesn't the `conversation-service` module handle interfacing with `smooch-core`? This one seems to just hold helper methods